docs/src/pages/[platform]/connected-components/in-app-messaging/examples/custom-components.react-native.mdx

```diff
@@ -0,0 +1,73 @@
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+import { Example, ExampleCode } from '@/components/Example';
+
+<Example>
+<ExampleCode>
+```jsx
+import React, { useCallback, useEffect } from 'react';
+import { Button, Text, View } from 'react-native';
+
+import { Amplify, Notifications } from 'aws-amplify';
+import {
+  useInAppMessaging,
+  withInAppMessaging,
+} from '@aws-amplify/ui-react-native';
```

Comment:
Until we have worked through the handling of multiple `Amplify.configure` calls in the React Native example app, we write code examples directly in the docs for React Native